ext/opcache/jit/zend_jit_vm_helpers.c

```diff
@@ -1143,10 +1119,6 @@ zend_jit_trace_stop ZEND_FASTCALL zend_jit_trace_execute(zend_execute_data  *ex,
 					}
 				}
 
-#ifndef HAVE_GCC_GLOBAL_REGS
-				opline = EX(opline);
-#endif
-
```

Comment:
Will this work with other VM types (GOTO and SWITCH)?

Reply:
I will check!

Reply:
JIT doesn't support the GOTO and SWITCH VMs currently: https://github.com/php/php-src/blob/f75dd82866382bf5511193a77d8391639652607f/ext/opcache/jit/zend_jit.c#L3677

I'm assuming it's because `zend_get_opcode_handler_func()` can not be implemented, as there are no opcode handler functions: https://github.com/php/php-src/blob/f75dd82866382bf5511193a77d8391639652607f/Zend/zend_vm_execute.skl#L126

We could support these VM kinds in the future by generating handler funcs in addition to the big SWITCH/GOTO. If these functions use the same conventions as the CALL VM, then the changes in this PR will work with these VM kinds. WDYT?